addons/web/static/src/views/fields/float_time/float_time_field.xml

```diff
@@ -3,7 +3,7 @@
 
     <t t-name="web.FloatTimeField" owl="1">
         <span t-if="props.readonly" t-esc="formattedValue" />
-        <input t-else="" t-att-id="props.id" t-ref="numpadDecimal" t-att-placeholder="props.placeholder" inputmode="numeric" class="o_input" />
+        <input t-else="" t-att-id="props.id" t-ref="numpadDecimal" t-att-placeholder="props.placeholder" inputmode="numeric" type="text" class="o_input" />
```

Comment:
Laurent has a issue because of that, but I don't remember why. However, since I no longer used that component since `record` prop is now required, I will remove that commit.